network/src/commonMain/kotlin/com/wire/kalium/network/utils/ObfuscateUtil.kt

```diff
@@ -5,16 +5,9 @@ package com.wire.kalium.network.utils
 import com.wire.kalium.logger.obfuscateDomain
 import com.wire.kalium.logger.obfuscateId
 import com.wire.kalium.logger.obfuscateUrlPath
-import io.ktor.http.Url
+import io.ktor.http.*
 import kotlinx.serialization.decodeFromString
-import kotlinx.serialization.json.Json
-import kotlinx.serialization.json.JsonArray
-import kotlinx.serialization.json.JsonElement
-import kotlinx.serialization.json.JsonNull
-import kotlinx.serialization.json.JsonObject
-import kotlinx.serialization.json.JsonPrimitive
-import kotlinx.serialization.json.jsonArray
-import kotlinx.serialization.json.jsonObject
+import kotlinx.serialization.json.*
```

Comment:
IDE is 🥴 
we shouldn't use the `.*`

Reply:
yeah mi IDE got it wrong with auto format thanks 🙌